lambda/security-groups/src/main/scala/com/gu/hq/Notifier.scala

```diff
@@ -40,7 +40,7 @@ object Notifier extends StrictLogging {
          |Warning: Security group **$groupId** in account **$accountName** is open to the world.
          |
          |The security group has ${targetTags.length} tags.
-         |${targetTags.map(t => s"**${t.key}**: ${t.value}").mkString(", ")}
+         |${targetTags.map(t => s"**${t.key}**: ${t.value}").mkString("\n")}
```

Comment:
Does it need to be "|\n"? I feel like the answer to this should be obvious but it doesn't sound it to me this afternoon 🤔

Reply:
Possibly! I was mildly surprised that no tests failed as a result of this change so I'll stick one in and see what happens.

Reply:
Turns out it's totally happy with `\n`. `\n|` would also have been fine, but `|\n` causes the test to fail.